core/src/decl/Kokkos_Declare_SYCL.hpp

```diff
@@ -35,6 +35,16 @@
 #include <SYCL/Kokkos_SYCL_ParallelScan_Range.hpp>
 #include <SYCL/Kokkos_SYCL_UniqueToken.hpp>
 #include <SYCL/Kokkos_SYCL_ZeroMemset.hpp>
+/*
+namespace Kokkos {
+namespace Experimental {
+using SYCLDeviceUSMSpace           = ::Kokkos::SYCLDeviceUSMSpace;
+using SYCLHostUSMSpace = ::Kokkos::SYCLHostUSMSpace;
+using SYCLSharedUSMSpace    = ::Kokkos::SYCLSharedUSMSpace;
+using SYCL                = ::Kokkos::SYCL;
+}  // namespace Experimental
+}  // namespace Kokkos
+*/
```

Comment:
Did you define these aliases elsewhere?

Reply:
They were not meant tobe  commented here. I checked before that defining these aliases and not changing anything in the tests would work.